### PR TITLE
What's New in 24.2 beta

### PR DESCRIPTION
**Pull request by micheleRP**
## Description

Resolves https://github.com/redpanda-data/documentation-private/issues/2557

## Page previews
https://deploy-preview-571--redpanda-docs-preview.netlify.app/24.2/get-started/whats-new/

## Checks

- [X] New feature
- [ ] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

### CODE REVIEW

File: modules/get-started/pages/whats-new.adoc [hunk 1]
```diff
@@ -2,76 +2,23 @@
...
+Redpanda's xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry] now supports JSON serialization format, in addition to Avro and Protobuf. A serialization format defines how data is converted into bytes that are transmitted and stored.
 
-== RBAC
-xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). Using `rpk` or Redpanda Console, you can create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those privileges. RBAC provides a more flexible and efficient way to manage user privileges, especially with complex organizational hierarchies or large numbers of users.  
+== Developer guide for data transforms
```
Comment (JakeSCahill) [inline modules/get-started/pages/whats-new.adoc]:
```suggestion
== Data transforms

You can now deploy transform functions that xref:develop:data-transforms/deploy.adoc#reprocess[reprocess existing records] in an input topic. Processing existing records can be useful if, for example, the output topic was accidentally deleted, and you need to recreate the lost data by reprocessing those existing records.
```

File: modules/get-started/pages/whats-new.adoc [hunk 2]
```diff
@@ -2,76 +2,23 @@
...
-
-== Documentation enhancements
-
-* https://docs.redpanda.com/redpanda-labs/[Redpanda Labs] provides examples, guidance, best practices, and ideas for how to use Redpanda in your own project.
-* xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect] documentation is added as a new dropdown option, similar to Redpanda Labs.
-* Improved search lets you filter for blogs, videos, labs, and documentation for all supported versions.  
-* New xref:upgrade:migrate/kubernetes/strimzi.adoc#migrate-kafka[side-by-side codeblocks] let you easily compare code examples.
-* Improved listing of changes to `rpk` commands in xref:upgrade:deprecated/index.adoc[deprecated features].
-* List of xref:reference:partner-integration.adoc[Redpanda integrations] built and supported by our partners.
+The docs now include an xref:develop:data-transforms/index.adoc[expanded guide] designed to help you master the creation, deployment, and management of data transforms in Redpanda.
```
Comment (JakeSCahill) [inline modules/get-started/pages/whats-new.adoc]:
```suggestion
The docs now also include an xref:develop:data-transforms/index.adoc[expanded guide] designed to help you master the creation, deployment, and management of data transforms in Redpanda.
```

**Review by Feediver1 — APPROVED** [review]
lgtm

File: modules/get-started/pages/whats-new.adoc [hunk 3]
```diff
@@ -2,76 +2,25 @@
...
-
-== RBAC
-xref:manage:security/authorization/rbac.adoc[Role-based access control] (RBAC) provides an extension to Redpanda's existing access control lists (ACLs). Using `rpk` or Redpanda Console, you can create roles and assign those roles as the authorized or denied party for an ACL. Every user you assign to that role inherits those privileges. RBAC provides a more flexible and efficient way to manage user privileges, especially with complex organizational hierarchies or large numbers of users.  
+Redpanda's xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry] now supports JSON serialization format, in addition to Avro and Protobuf. A serialization format defines how data is converted into bytes that are transmitted and stored.
```
Comment (kbatuigas) [inline modules/get-started/pages/whats-new.adoc]:
```suggestion
Redpanda's xref:manage:schema-reg/schema-reg-overview.adoc[Schema Registry] now supports JSON serialization format, in addition to Avro and Protobuf. You can use a JSON Schema format to validate messages and ensure compatibility between schema versions.
```

**Review by kbatuigas — COMMENTED** [review]
Just a suggestion but I'd defer to @mattschumpert ultimately on the content here.

File: modules/get-started/pages/whats-new.adoc (hunk 3 above)
Comment (mattschumpert) [reply modules/get-started/pages/whats-new.adoc]: @kbatuigas probably we want to say specifically "JSON Schema" and link here: https://json-schema.org/

eh @BenPope ?

File: modules/get-started/pages/whats-new.adoc [hunk 4]
```diff
@@ -2,76 +2,25 @@
...
-- xref:reference:rpk/rpk-cluster/rpk-cluster-partitions-transfer-leadership.adoc[`rpk cluster partitions transfer-leadership`]
-- xref:reference:rpk/rpk-redpanda/rpk-redpanda-tune-help.adoc[`rpk redpanda tune help`]
-- xref:reference:rpk/rpk-transform/rpk-transform-logs.adoc[`rpk transform logs`]
-- xref:reference:rpk/rpk-connect/rpk-connect.adoc[`rpk connect` commands]
-- xref:reference:rpk/rpk-security/rpk-security.adoc[`rpk security` commands]
-+
-Additionally, to bundle security commands together, the existing `rpk acl` commands have moved into the new `rpk security` section as xref:reference:rpk/rpk-security/rpk-security-acl.adoc[`rpk security acl`]. The existing `rpk acl` commands still work as an alias.
-
-== Documentation enhancements
+You can now deploy transform functions that xref:develop:data-transforms/deploy.adoc#reprocess[reprocess existing records] in an input topic. Processing existing records can be useful if, for example, the output topic was accidentally deleted, and you need to recreate the lost data by reprocessing those existing records.
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new.adoc]: @kbatuigas 

"You can now deploy Data Transform functions that xref:develop:data-transforms/deploy.adoc#reprocess[reprocess existing records] from an input topic. Processing existing records can be useful, for example, to process historical data into a different format for a new Consumer, or if a topic was accidentally deleted and you need to recreate the lost data by reprocessing those existing records, or to resolve issues with a previous version of a transform that processed data incorrectly.

@oleiman  WDYT?  

Once we agree we could also use that verbiage here: https://deploy-preview-571--redpanda-docs-preview.netlify.app/24.2/develop/data-transforms/deploy/#reprocess

File: modules/get-started/pages/whats-new.adoc [hunk 5]
```diff
@@ -2,76 +2,25 @@
...
+This topic includes new content added in version 24.2 beta. For a complete list of all product updates, see the https://github.com/redpanda-data/redpanda/releases/[Redpanda release notes^]. See also: 
 
 * xref:deploy:deployment-option/cloud/whats-new-cloud.adoc[] 
 * xref:deploy:deployment-option/cloud/cloud-overview.adoc#redpanda-cloud-vs-self-hosted-feature-compatibility[Redpanda Cloud vs self-hosted feature compatibility]
 
-== Redpanda Connect 
+== FIPS compliance
 
-In release 24.1.5, Redpanda announced xref:redpanda-connect:ROOT:about.adoc[Redpanda Connect]. This includes over 220 open source connectors, with a large collection of sources, sinks, and processors. The interface provides a lightweight and easy way to set up, debug, and manage connectors. 
+Redpanda now provides FIPS-compliant cipher enforcement for brokers using OpenSSL. Both Redpanda and `rpk` leverage validated OpenSSL libraries for all security-related cryptography operations. Install the `redpanda-fips` package to xref:manage:security/fips-compliance.adoc[run Redpanda in FIPS compliance mode].
```
Comment (mattschumpert) [inline modules/get-started/pages/whats-new.adoc]: @kbatuigas 

let's go with:

"Redpanda now provides a FIPS 140-compliant binary to support running Redpanda in environments requiring FIPS-compliant software, and to meet certain requirements for users undergoing certification through government compliance programs such as FedRAMP.  When using the FIPS binary, both Redpanda and `rpk` leverage FIPS-compliant OpenSSL libraries and ciphers for all encryption activity inside Redpanda, including encryption of data in transit.  Install the `redpanda-fips` package to run Redpanda in FIPS-compliance mode."

**Review by mattschumpert — CHANGES_REQUESTED** [review]

File: modules/get-started/pages/whats-new.adoc (hunk 3 above)
Comment (mattschumpert) [reply modules/get-started/pages/whats-new.adoc]: the suggestion second sentence sgtm

File: modules/get-started/pages/whats-new.adoc (hunk 4 above)
Comment (oleiman) [reply modules/get-started/pages/whats-new.adoc]: lgtm. accurate and informative 🙂

File: modules/get-started/pages/whats-new.adoc (hunk 5 above)
Comment (mattschumpert) [reply modules/get-started/pages/whats-new.adoc]: @kbatuigas  on second thought, 'users' should be 'organizations'.   Besides that looks good

**Review by mattschumpert — APPROVED** [review]

**Pull request merged**